pybullet_rendering/examples/panda3d_gui.py

```diff
@@ -163,37 +168,30 @@ def update_scene(self, scene_graph, materials_only):
             node = self.render.attachNewNode(f'node_{k:02d}')
             self.nodes[k] = node
 
-            for j, shape in enumerate(v.shapes):
-                if shape.type == ShapeType.Mesh:
-                    filename = shape.mesh.filename
-                elif shape.type == ShapeType.Cube:
-                    filename = 'cube.obj'
-                else:
-                    print('Unknown shape type: {}'.format(shape.type))
+            for j, pb_shape in enumerate(v.shapes):
+                filename = shape_filename(pb_shape)
```

Comment:
We use `shape_filename` from `pybullet_rendering.render.utils` to avoid duplicating code. It's also what's used in the panda3d renderer file